templates/Includes/CommentPagination.ss

```diff
@@ -12,7 +12,7 @@
                     <% loop $Pages %>
                         <% if $CurrentBool %>
                             <li class="page-item active">
-                                <a class="page-link"><strong>$PageNum</strong></a>
+                                <a class="page-link" href="#"><strong>$PageNum</strong></a>
```

Comment:
Are the events already cancelled on these links? Perhaps flicking the user to the top of the page is expected?

Reply:
The alternative would be to use a `span`, events are not disabled by this